datamade/how-to
# Initialize project management documentation

## Overview

Add some basic documentation for project management, including:

- Project collateral
- Best practices for working with source control

Closes https://github.com/datamade/how-to/issues/20.

## Testing instructions

- View the rendered directory on GitHub: https://github.com/datamade/how-to/tree/feature/jfc/project-management/project-management
- Confirm that the markdown looks good (including links)
